src/index.js

```diff
@@ -187,6 +187,8 @@ class Node extends EventEmitter {
           return each(this.modules.discovery, (d, cb) => d.start(cb), cb)
         }
         cb()
+
+        this.emit('online')
```

Comment:
Perhaps we should use the same verbs, otherwise it may get confusing to users, no?
The other alternative is to be explicit (in the docs) about the state machine (states) and the transitions (events). Example:

starting event -> starting state -> started event -> running state -> stopping event -> stopped state

Makes sense?